src/lib/ui/app/Alerts/form-steps/metric-conditions/utils.ts

```diff
@@ -0,0 +1,36 @@
+import type { TConditionsState } from './schema.js'
+
+import { parseRangeString } from '$lib/utils/dates/index.js'
+
+import { TimeModifiers } from '../../time.js'
+import { Operations, type TOperationType } from './operations.js'
+
+function checkIsUsdMetric(metric: string) {
+  return new Set(['price_usd', 'marketcap_usd']).has(metric)
```

Comment:
Potential optimization: 1) Instead of creating Set on every call, it could placed as a constant variable outside the function 2) Since array is small, calling `['price_usd', 'marketcap_usd'].includes(metric)` is also a valid optimization.